README.md

```diff
@@ -35,8 +41,11 @@ the Google Cloud Console](https://console.cloud.google.com/appengine/versions).
 
 ## Next steps
 
-1.  Read the [App Engine documentation](https://cloud.google.com/appengine/docs/java/).
-1.  Explore the other [Google Cloud Platform sample applications](https://github.com/GoogleCloudPlatform/java-docs-sample://github.com/GoogleCloudPlatform/java-docs-samples) for Java™.
+1.  Read the [App Engine
+    documentation](https://cloud.google.com/appengine/docs/java/).
+1.  Explore the other [Google Cloud Platform sample
+    applications](https://github.com/GoogleCloudPlatform/java-docs-sample) for
+    Java™.
```

Comment:
Might be a good idea. I wonder if we can leverage the auto-readme-gen that python-docs-samples is using.